pkg/config/config.go

```diff
@@ -29,6 +33,7 @@ const (
 
 type Config struct {
 	DataDir          string
+	LogDir           string
```

Comment:
This name is misleading. Actually it is the directory to store searched log results. How about calling it `TempStoreDir` and use it to replace the `ioutil.TempDir`? (remember to change the cli flag name as well)

Reply:
Sorry for the later reply, I'll change it later today.